src/runtime.c

```diff
@@ -361,7 +361,9 @@ bool extract_appimage(const char* const appimage_path, const char* const _prefix
                     } else {
                         struct stat st;
                         if (!overwrite && stat(prefixed_path_to_extract, &st) == 0 && st.st_size == inode.xtra.reg.file_size) {
-                            fprintf(stderr, "File exists and file size matches, skipping\n");
+                            if (verbose) {
+                                fprintf(stderr, "File exists and file size matches, skipping\n");
```

Comment:
This is the only place where I am checking for verbose flag. The other `fprintf`s were being called before a `return` so I thought it'd be unwise for `verbose == False` to block them. Can add in if required.